openshift/templates/rails-postgresql.json

```diff
@@ -202,10 +202,6 @@
                     "value": "${POSTGRESQL_SHARED_BUFFERS}"
                   },
                   {
-                    "name": "SECRET_KEY_BASE",
```

Comment:
this is already defined in https://github.com/openshift/rails-ex/pull/43/files#diff-3b357a8da1a0210bb9d058811f39a903R193